pkgs/by-name/po/powermanga/package.nix

```diff
@@ -21,6 +22,9 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     autoconf
     automake
+    # needed as of https://github.com/libsdl-org/SDL-1.2/commit/fa041e14b5f69a820b0250ae7f450111af2fdbba
+    # which emits a PKG_CHECK_MODULES(SDL, ...) statement which sh cannot parse
+    pkg-config
```

Comment:
pkg-config is a relatively normal dependency to see. Commenting on it is fine, but probably wasn't necessary

Reply:
(wrong button, full review coming in a bit... wanted to hit "start a review", i hate github ui)

Reply:
I opted to comment since this seems to be a breaking change in SDL1?